src/Symfony/Component/ExpressionLanguage/ExpressionFunction.php

```diff
@@ -62,4 +62,46 @@ public function getEvaluator()
     {
         return $this->evaluator;
     }
+
+    /**
+     * Creates an ExpressionFunction from a PHP function name.
+     *
+     * @param string      $phpFunction        The PHP function name
+     * @param string|null $expressionFunction The expression function name (default: same than the PHP function name)
+     *
+     * @return self
+     *
+     * @throws \InvalidArgumentException if given PHP function name does not exist
+     * @throws \InvalidArgumentException if given PHP function name is in namespace
+     *                                   and expression function name is not defined
+     */
+    public static function fromPhp($phpFunction, $expressionFunction = null)
+    {
+        if (!function_exists($phpFunction)) {
+            throw new \InvalidArgumentException(sprintf('PHP function "%s" does not exist.', $phpFunction));
+        }
+
+        $reflection = new \ReflectionFunction($phpFunction);
+        if ($reflection->inNamespace() && !$expressionFunction) {
```

Comment:
Instead of using reflection, what about just checking for `\\` in the function name?